README.md

```diff
@@ -93,6 +93,44 @@ Have a look at [Contributing.md](Contributing.md) for the full details on the
 Conventional Commit messages we prefer, our code formatting conventions, our Git
 hooks, and recommendations on how to make effective pull requests.
 
+## Publishing Instructions
+
+To publish a new version of the project, follow these steps:
+
+1. **Determine the Version Bump**
+   Decide whether your changes constitute a:
+   - **Major** version bump (breaking changes),
+   - **Minor** version bump (new features, backward-compatible, any change to the schema), or
+   - **Patch** version bump (bug fixes, small changes).
+
+2. **Update Version Files**
+   - Update the `everyvoice._version` file to reflect the new version.
+   - Keep all `submodule._version` files in sync with this version, **except** for the `wav2vec2` aligner submodule (which can be installed separately).
+   - Commit the resulting changes, including all submodules.
+
+3. **Update Schema (for Major/Minor bumps)**
+   If you bumped a **major** or **minor** version:
+   - Run `everyvoice update-schema`.
```

Comment:
I would add `(You may need to delete some existing schema files if you get an error message.)` or something to that effect. Our process is not actually obvious except to you and I, because you just get an error message that effectively says "no can do" without giving guidance or fixing it. -- But maybe we want to keep the group of people doing this small?

Reply:
Yes, you're right, but that doesn't mean we have to keep the documentation obscure. I just added a ruleset that prevents non-admin's from pushing v* tags.